projects/core/src/routing/store/reducers/router.reducer.spec.ts

```diff
@@ -199,4 +215,32 @@ describe('Router Reducer', () => {
       cmsRequired: false
     });
   });
+
+  describe('should set correct context for content pages', () => {
+    let context;
+
+    beforeEach(async () => {
+      store.subscribe(routerStore => {
+        context = routerStore.router.state.context;
+      });
+    });
+
+    it('for generic page', async () => {
+      await zone.run(() => router.navigateByUrl('/customCmsPage'));
```

Comment:
Why inside zone.run?

Reply:
To suppress warnings from router module in tests